subgraphs/bridgeworld/tests/summoning.test.ts

```diff
@@ -29,34 +39,71 @@ test("summon is started and finished with result token", () => {
   const randomRequestEvent = createRandomRequestEvent(0, 1);
   handleRandomRequest(randomRequestEvent);
 
-  const mintEvent = createLegionTransferEvent(
+  let mintEvent = createLegionTransferEvent(
     Address.zero().toHexString(),
     USER_ADDRESS,
     7
   );
+
   handleTransfer(mintEvent);
 
-  const legionCreatedEvent = createLegionCreatedEvent(USER_ADDRESS, 7, 0, 6, 2);
+  mintEvent = createLegionTransferEvent(
+    Address.zero().toHexString(),
+    USER_ADDRESS,
+    3822
+  );
+
+  handleTransfer(mintEvent);
+
+  let legionCreatedEvent = createLegionCreatedEvent(USER_ADDRESS, 7, 0, 6, 2);
+
+  handleLegionCreated(legionCreatedEvent);
+
+  legionCreatedEvent = createLegionCreatedEvent(USER_ADDRESS, 3822, 0, 6, 2);
+
   handleLegionCreated(legionCreatedEvent);
 
   const summoningStartedEvent = createSummoningStartedEvent(
     USER_ADDRESS,
     7,
     0,
-    1643659676
+    1643659676,
+    [7, 3822],
+    [2, 0]
   );
   handleSummoningStarted(summoningStartedEvent);
 
   const id = `${LEGION_ADDRESS.toHexString()}-0x7`;
   assert.fieldEquals(TOKEN_ENTITY_TYPE, id, "category", "Legion");
 
-  const metadata = `${id}-metadata`;
-  assert.fieldEquals(LEGION_INFO_ENTITY_TYPE, metadata, "summons", "1");
-
   const summon = `${summoningStartedEvent.address.toHexString()}-0x7`;
   assert.fieldEquals(SUMMON_ENTITY_TYPE, summon, "token", id);
   assert.fieldEquals(SUMMON_ENTITY_TYPE, summon, "status", "Idle");
 
+  createMockedFunction(
+    SUMMONING_ADDRESS,
+    "didSummoningSucceed",
+    "didSummoningSucceed(uint256):(bool,uint256)"
+  )
+    .withArgs([ethereum.Value.fromI32(7)])
+    .returns([
+      ethereum.Value.fromBoolean(true),
+      ethereum.Value.fromI32(1644659676),
+    ]);
+
+  const seededEvent = createRandomSeededEvent(1);
+
+  handleRandomSeeded(seededEvent);
+
+  assert.fieldEquals(SUMMONING_CIRCLE_ENTITY_TYPE, "only", "crafters", "0");
+  assert.fieldEquals(SUMMONING_CIRCLE_ENTITY_TYPE, "only", "summoners", "1");
+  assert.fieldEquals(
+    SUMMONING_CIRCLE_ENTITY_TYPE,
+    "only",
+    "successRate",
+    "0.00008"
```

Comment:
Should be 0.00001 in this case? Contract has this check:
```
if(_numberOfCraftings == 0) {
  return 1;
}
```

Reply:
Math is correct now. :)